### ISSUE TITLE
<br> not rendering

### ISSUE THREAD

**Issue by dGilli** [verbb/vizy#54]
**Description**
Hard breaks "<br>" not rendering.

**Steps to reproduce**
1. Insert hard break with "shift + enter" in vizy field.
2. <br> should be rendered as line break.

**Additional info**
- Plugin version: 1.0.3
- Craft version: 3.6.17
- Multi-site: Yes

**Additional context**
The break is present in the fields html view and when outputting via `{{ dump(entry.vizyField.getRawNodes()) }}`:

```
array(1) {
    ["type"]=>
    string(9) "hardBreak"
}
```

When rendered with `.renderHtml()` the break is no there.

**Comment by adrienpicard** [comment]
Having the same issue with a `<hr>` tag. Possibly related to all self closing tags? Noticed that [renderNode](https://github.com/verbb/vizy/blob/dc68f50efad565a53c5c205f484b2d793d559812/src/services/Nodes.php#L139) breaks out when it's the case.

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.4](https://github.com/verbb/vizy/releases/tag/1.0.4)
